hadoop-ozone/dist/src/shell/conf/log4j.properties

```diff
@@ -59,16 +59,32 @@ log4j.appender.ASYNCRFA.conversionPattern=%d{ISO8601} [%t] %p %c: %m%n
 # Daily Rolling File Appender
 #
 
-log4j.appender.DRFA=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.DRFA.File=${hadoop.log.dir}/${hadoop.log.file}
+log4j.appender.DRFA=org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.DRFA.rollingPolicy=org.apache.log4j.rolling.FixedWindowRollingPolicy
+log4j.appender.DRFA.triggeringPolicy=org.apache.log4j.rolling.SizeBasedTriggeringPolicy
 
-# Rollover at midnight
-log4j.appender.DRFA.DatePattern=.yyyy-MM-dd
+log4j.appender.DRFA.rollingPolicy.ActiveFileName=${hadoop.log.dir}/${hadoop.log.file}
+log4j.appender.DRFA.rollingPolicy.FileNamePattern=${hadoop.log.dir}/${hadoop.log.file}-%i.log.gz
+#MaxFileSize param accepts a long value
+log4j.appender.DRFA.triggeringPolicy.MaxFileSize=268435456
+log4j.appender.DRFA.rollingPolicy.maxIndex=10
```

Comment:
This'll retain up to 10 256MB compressed log files. That should be good enough for most cases.